style.css

```diff
@@ -806,11 +788,321 @@ header {
 }
 
 .fmargin {
-  position: absolute;
+  order: 5;
   width: 134px;
   height: 5px;
   left: calc(50% - 134px / 2 - 0.5px);
   bottom: 8px;
   border-radius: 10px;
   background-image: url("./Images/Shape.png");
 }
+
+/* media query section */
+
+@media (min-width: 768px) {
+  header {
+    width: 100%;
+    height: 72px;
+  }
+
+  .bar2 {
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .bar2 .logo {
+    width: 116px;
+    height: 44px;
+    left: 142px;
+    top: 16px;
+    gap: 10px;
+  }
+
+  .bar2 .normalb {
+    display: none;
+  }
+
+  .bar2 .burger {
+    margin-right: 6rem;
+    display: flex;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    background-color: #3c3a39;
+    border: none;
+    align-items: center;
+    top: 16px;
+  }
+
+  .bar2 .burger a {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 10px 12px;
+    gap: 10px;
+    width: 15%;
+    height: 40px;
+    flex: none;
+    flex-grow: 0;
+  }
```

Comment:
- [ ] Kindly add some margins between the nav links, presently, they are overlapping each other.
Your design
![Capture](https://user-images.githubusercontent.com/85095702/171738073-a29c69e4-f240-4616-bc39-3678191133e6.jpg)